Makefile

```diff
@@ -40,7 +40,7 @@ docker: build
 endif
 	docker build --tag "${DOCKER_REGISTRY}/pingcap/tidb-operator:${IMAGE_TAG}" images/tidb-operator
 
-build: controller-manager scheduler discovery admission-controller apiserver backup-manager
+build: controller-manager scheduler discovery aggragated-webhook apiserver backup-manager
```

Comment:
I think this is still an admission webhooks, `aggragated-webhook ` is strange.

Reply:
updated